colibris/authentication/__init__.py

```diff
@@ -7,7 +7,7 @@
 from .exceptions import *
 
 
-_REQUEST_ACCOUNT_ITEM_NAME = 'account'
+REQUEST_ACCOUNT_KEY = 'account'
```

Comment:
@ccrisan, i think the "KEY" version is more clear about the purpose of that constant. I renamed them all using this pattern, and made them all public. Is it fine this way?

Reply:
Yes, I like it this way.